numpy/ma/core.pyi

```diff
@@ -14,9 +14,12 @@ from numpy import (
     zeros_like as zeros_like,
     angle as angle
 )
+from numpy._typing import _ArrayLike, _ScalarLike_co, _ShapeLike, overload
 
 _ShapeType_co = TypeVar("_ShapeType_co", bound=tuple[int, ...], covariant=True)
 _DType_co = TypeVar("_DType_co", bound=dtype[Any], covariant=True)
+_ArrayType = TypeVar("_ArrayType", bound="MaskedArray[Any, Any]")
```

Comment:
There's no need for stringified annotations within `.pyi` stubs (and disallowed by e.g. `flake8-pyi` and several type-checkers)